.nvmrc

```diff
@@ -1 +1 @@
-16
+18
```

Comment:
is it the version on our servers?

Reply:
It's a browser extension, there are no servers :)

Reply:
Or if you mean the CI servers, then yes - [I made sure that CI sources the node version from this file](https://github.com/Cardinal-Cryptography/aleph-zero-signer/pull/136/files#diff-e1d72902548ef5b95cfcd2adc5fe403bbf212593b41b3eb8f564617cc8a54380R15-R18).